documentation/proc-pages/physics-models/plasma_current/plasma_current.md

```diff
@@ -4,21 +4,32 @@
 
 In tokamaks, the plasma current ($I_{\text{p}}$) plays a crucial role in confining the plasma and maintaining the stability of the magnetic fields. The plasma current generates a poloidal magnetic field, which, when combined with the toroidal magnetic field, creates a helical magnetic field structure. 
 
-The plasma current is typically driven by inductive means, where a central solenoid induces a current in the plasma, similar to the secondary winding of a transformer. However, non-inductive methods such as neutral beam injection and radiofrequency (RF) heating are also employed, especially for steady-state operation.
+The plasma current is typically driven by inductive means, where a central solenoid induces a current in the plasma, similar to the secondary winding of a transformer. However, non-inductive methods such as neutral beam injection and radio frequency (RF) heating are also employed, especially for steady-state operation.
+
+More information about the different heating and current drive models can be found [here](../../eng-models/heating_and_current_drive/heating-and-current-drive.md).
+
+In `PROCESS` there are several inductive and non-inductive contributions to the overall plasma current. The fractions of these contributions are checked to add up to 1.0.
+
+$$
+1.0 = [\underbrace{\underbrace{I_{\text{F,Boot}} + I_{\text{F,Dia}} + I_{\text{F,PS}}}_{\text{Generated by plasma}} + I_{\text{F,CD}}}_{\text{Non-inductive}}] + \underbrace{I_{\text{F,CS}}}_{\text{Inductive}}
```

Comment:
It's only a minor thing but it would possibly look better if the "generated by plasma" bracket was above the equation.

Reply:
Changed